site/src/contexts/ProxyContext.tsx

```diff
@@ -94,37 +95,8 @@ export const ProxyProvider: FC<PropsWithChildren> = ({ children }) => {
     computeUsableURLS(userSavedProxy),
   );
 
-  const queryKey = ["get-proxies"];
-  // This doesn't seem like an idiomatic way to get react-query to use the
-  // initial data without performing an API request on mount, but it works.
-  //
-  // If anyone would like to clean this up in the future, it should seed data
-  // from the `meta` tag if it exists, and not fetch the regions route.
-  const [initialData] = useState(() => {
-    // Build info is injected by the Coder server into the HTML document.
-    const regions = document.querySelector("meta[property=regions]");
-    if (regions) {
-      const rawContent = regions.getAttribute("content");
-      try {
-        const obj = JSON.parse(rawContent as string);
-        if ("regions" in obj) {
-          return obj.regions as Region[];
-        }
-        return obj as Region[];
```

Comment:
@Emyrk I wasn't completely sure why we had a check for whether the data getting parsed was either an array or an object with the array inside it, but just to be on the safe side, I ripped this out and centralized it in the metadata manager file

Reply:
Great question. The type safety we have on these fields is terrible, and should be resolved imo to make things like this more clear. We just inject strings that are json. Very weak imo.

I think it should always have `regions` as a field, but would have to do some digging to really confirm.